apps/common/src/python/mediawords/languages/lt/lt_stop_words.txt

```diff
@@ -1,19 +1,78 @@
-#
-# This is a stop word list for the Lithuanian language.
-#
 # Sources:
+#
 #     http://www.filewatcher.com/p/punbb-1.2.16.tbz.620109/www/punbb/lang/Lithuanian/stopwords.txt.html
+#     https://github.com/stopwords-iso/stopwords-lt/blob/master/stopwords-lt.txt
 #     auto-generated sources
 #
+#
 
 a
+á
+abi
+abidvi
+abiejø
+abiejose
+abiejuose
+abiem
+abigaliai
+abipus
+abu
+abudu
+ai
+ákypai
+ana
+anaiptol
+anaisiais
+anàja
+anàjá
+anàjà
+anajai
+anajam
+anajame
+anapus
+anas
+anasai
+anàsias
+anasis
+anei
+aniedvi
+anieji
+aniesiems
+anoji
+anojo
+anøjø
+anojoje
+anokia
+anoks
+anosiomis
+anosioms
+anosios
+anosiose
+anot
+ant
+antai
+anuodu
+anuoju
+anuosiuose
+anuosius
 apie
+aplink
 ar
 arba
+argi
+arti
+ástriþai
+aukðèiau
```

Comment:
Here and elsewhere, you didn't get the character set right :) The list that you've looked at was probably in ISO-8859-13, and this list needs to be in UTF-8. In other words, `aukðèiau` should have been `aukščiau`.